Features/Components/Component.php

```diff
@@ -21,7 +21,8 @@ public static function enqueueAssets($componentName, array $dependencies = [])
                 array_push($list, $dependency['name']);
             }
             return $list;
-        }, ['jquery']); // jquery as a default dependency
+        }, ['jquery-flynt']); // jquery-flynt as a default dependency
```

Comment:
Can this be in the DocumentDefault as well, so everything is in one place and easy to follow? I might not look and expect this in here.